src/atoms/index.ts

```diff
@@ -1,3 +1,3 @@
-export { default as Header } from './Header'
-export { default as Text } from './Text'
+export { default as Header, HeaderProps } from './Header'
+export { default as Text, TextProps } from './Text'
 export { default as Icon } from './Icon'
```

Comment:
Going forward I think we should always export component props as well. This way you can extend the props in citrus